packs/classfeatures/the-tangible-dream.json

```diff
@@ -11,7 +11,7 @@
ts. Choose an unoccupied square within the spell's range. The illusion in that square provides flanking for a single melee attack made before the beginning of your next turn. If you Sustain the spell, the details of the illusion change and shift to keep your enemies unsettled; the flanking illusion's duration extends until the beginning of your next turn and you can move it to any unoccupied square in the spell's range. The flanking illusion can't provide its benefit against any creature who has disbelieved the figment.</p>\n<p>@UUID[Compendium.pf2e.spells-srd.Item.Shield]</p>\n<p>Your <em>shield</em> spell can be deployed on behalf of others. When you cast <em>shield</em>, you can choose a target ally within 30 feet to gain the benefits of the spell instead of you. The option to Shield Block with the spell is up to you and uses your reaction, not the target's. Your <em>shield</em> also gains the following amp.</p>\n<p><strong>Amp</strong> You create a layered barrier of three shields to improve your defenses. The duration increases to sustained up to 1 minute. The shield still lowers at the start of each of your turns, but you can raise it again when you Sustain the Spell.</p>\n<p>When you Shield Block with this spell, one of the three layers breaks and you cease to gain the bonus to AC, but the spell doesn't end. You can keep using the spell until the final layer breaks; once it does, the spell ends and you can't cast shield or amped shield again for 10 minutes. You can choose to break additional layers when you Shield Block with this spell to improve your defenses. If you break two layers instead of one, increase the Hardness to 1-1/2 × the normal amount (7 at 1st level, 15 at 3rd level, 22 at 5th level, 30 at 7th level, and 37 at 9th level). If you break all three layers, increase the Hardness to double the normal amount.</p>\n<p><strong>Unique Psi Cantrips</strong></p>\n<p>surface: @UUID[Compendium.pf2e.spells-srd.Item.Imaginary Weapon]</p>\n<p><strong>Amp</strong> You form multiple force weapons to lash out at multiple foes. You make two imaginary weapon Strikes, each against a different target. Your multiple attack penalty doesn't increase until you've made both Strikes.</p>\n<p><strong>Amp Heightened (+1)</strong> The damage increases by 2d8 instead of 1d8.</p>\n<p>deeper: @UUID[Compendium.pf2e.spells-srd.Item.Astral Rain]</p>\n<p><strong>Amp</strong> The area changes to a @Template[burst|distance:5], and you can Sustain the Spell up to 1 minute. The first time each round that you Sustain the Spell, you can move the burst up to 20 feet.</p>\n<p>deepest: @UUID[Compendium.pf2e.spells-srd.Item.Hologram Cage]</p>\n<p><strong>Amp</strong> Force walls make the cage's barrier physical as well as visual. Each 10-foot-by-10-foot section of the cage's wall has AC 10, Hardness equal to double the spell's level, and HP equal to quadruple the spell's level. The spell is disrupted if any of its walls would pass through a creature when you cast it.</p>"
```

Comment:
No real changes here, just accidentally opened it on the editor and some html got automatically trimmed.